notesserver/docker_gunicorn_configuration.py

```diff
@@ -0,0 +1,45 @@
+"""
+gunicorn configuration file: http://docs.gunicorn.org/en/develop/configure.html
```

Comment:
Let's ship this, verify it, and move to gunicorn 20 ASAP :D

Reply:
agreed